### PR TITLE
Add action to audit publicly listed organizations

### PR DESCRIPTION
**Pull request by IanButterworth**
It seems to be reasonable to expect any publicly listed julia organization to have at least 2 public members.

This sets up an action to check that whenever the list is changed in `community/organizations.md`, or if the action is manually triggered, otherwise it won't run to avoid noise on unrelated PRs.

Screenshot 2023-09-29 at 2 25 16 PM

TODO:
- [x] Currently dependent on an unreleased version of GitHub.jl https://github.com/JuliaWeb/GitHub.jl/pull/218
- [x] Revert the `community/organizations.md` change before merge

### CODE REVIEW

File: .github/workflows/check_orgs.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+      - uses: julia-actions/setup-julia@v1
+        with:
+          version: 1.9
+
+      - uses: julia-actions/julia-buildpkg@v1
+        with:
+          project: tools/check_orgs
+
+      - run: julia check_orgs.jl
```
Comment (giordano) [inline .github/workflows/check_orgs.yml:25]: I guess you want to specify the project, no?
```suggestion
      - run: julia --project=. --color=yes check_orgs.jl
```

**Comment by StefanKarpinski** [comment]
Very good idea!

**Comment by IanButterworth** [comment]
@StefanKarpinski do you think `community/organizations.md` should also have some sort of notice like:
```
Organizations listed here should strive to have at least 2 members with admin rights, and at least 2 public members. 
Doing so helps to protect the organization's repositories against a single person being unreachable.
```

**Comment by DilumAluthge** [comment]
> at least 2 members with admin rights

It might be clearer to change "members with admin rights" to "owners".

**Comment by ViralBShah** [comment]
Looks great. Let's get it merged.

**Comment by IanButterworth** [comment]
Ok. What do you think about https://github.com/JuliaLang/www.julialang.org/pull/1941#issuecomment-1758076382

**Comment by ViralBShah** [comment]
> Ok. What do you think about [#1941 (comment)](https://github.com/JuliaLang/www.julialang.org/pull/1941#issuecomment-1758076382)

That feels totally reasonable. Let's do it.

**Comment by IanButterworth** [comment]
I'll go ahead with this. At this point, this is the status
```
Finding orgs with fewer than 2 public memers that are listed in https://julialang.org/community/organizations/
 - JuliaPraxis 1 members
 - JuliaActors 0 members
 - JuliaFirstOrder 1 members
 - JuliaNLSolvers 1 members
 - JuliaNonconvex 1 members
 - JuliaActuary 1 members
 - ModiaSim 1 members
 - JuliaAstrodynamics 1 members
 - juliahci 1 members
 - JuliaQuantum 1 members

```

**Pull request merged**